docs/topics/about-rules.adoc

```diff
@@ -19,7 +19,7 @@ when(condition)
  tag(tags)
 ----
 
-Although {ProductShortName} provides a comprehensive set of standard migration rules out-of-the-box, you can create your own custom analyzer rules as well. You can use custom rules to identify use of custom libraries or other components that might not be covered by the provided out-of-the-box rulesets.
+Although {ProductShortName} provides a comprehensive set of standard migration rules out-of-the-box, you can create your own custom analyzer rules as well. You can use custom rules to identify use of custom libraries or other components that might not be covered by the provided rulesets.
```

Comment:
suggestion (grammar) 
"You can use custom rules to identify _the_ use of custom libraries or other components that _are_ not covered by the provided rulesets."